src/components/charts/colorMixingTool/color-tool.module.scss

```diff
@@ -48,7 +48,6 @@
 
 .images-container {
   position: relative;
-  height: 300px;
   padding: $minPadding;
   overflow: hidden;
 }
```

Comment:
There is some cool aspect-ratio stuff you can do with the code to keep a specific height regardless of width. It looks something like this:
```
.your-class-name {
  width: 100%;
  padding-top: 100%; // for 1:1 ratio
}
```
You can find some doc stuffs about [aspect ratio here](https://www.w3schools.com/howto/howto_css_aspect_ratio.asp)